Sources/ParseSwift/Coding/AnyEncodable.swift

```diff
@@ -109,6 +109,8 @@ extension _AnyEncodable {
             throw EncodingError.invalidValue(self.value, context)
         }
     }
+
+    #if !os(Linux)
     private func encode(nsnumber: NSNumber, into container: inout SingleValueEncodingContainer) throws { // swiftlint:disable:this cyclomatic_complexity line_length
```

Comment:
Why does the encoder need special behavior for Linux vs. non-Linux platforms?

Reply:
Good question, Swift Linux doesn't like somethings from objective-c or is missing the bridging for them, such as `NSNumber`. These I don't know specifically while coding, I figure them out by looking at the Linux build for Parse-Swift on [Swift Package Index](https://swiftpackageindex.com/builds/1160B83E-99C1-44B6-8E0E-85FD8C5833EC) and searching online. This will bring the linux build closer to fruition, but I'm sure there are still some minor issues similar to this.

Reply:
I do the same for the other builds as Swift Package Index runs the unit tests for all of the builds (and then populates the badges on the Parse-Swift readme). For GitHub Actions, the test cases are only ran for iOS and SPM macOS. Builds are ran for the others in GitHub Actions, but not the unit tests.